app/services/images_for_tag_name.rb

```diff
@@ -1,10 +1,10 @@
 class ImagesForTagName
   def self.images(tag_name)
-    if tag_name.nil?
-      image_list = Image.order(created_at: :desc)
-    else
-      image_list = Image.tagged_with(tag_name).order(created_at: :desc)
-    end
+    image_list = if tag_name.nil?
```

Comment:
We usually don't prefer a commit for fix code styles. We will fixup to the commit which this belongs to.